.github/workflows/dependencies_check.yml

```diff
@@ -0,0 +1,26 @@
+name: Dependencies Check
```

Comment:
Is this necessary? Unless this also checks lua dependencies it's a wast of resources.

Reply:
Removed.